### PR TITLE
Make README nicer and more correct...

### PR DESCRIPTION
**Pull request by destogl**
I will also update description.

### CODE REVIEW

**Comment by bmagyar** [comment]
The branch name for foxy is master atm

**Comment by bmagyar** [comment]
also, is this a draft by accident? I'd be happy to merge...

**Comment by destogl** [comment]
> also, is this a draft by accident? I'd be happy to merge...

It's intentional. I wanted to do more changes. Give me time until tomorrow, please

**Comment by bmagyar** [comment]
ping

**Comment by destogl** [comment]
@bmagyar @v-lopez can we be more precise in Acknowledgement? e.g., by stating which parts are implemented using ROSIN funding?

**Comment by bmagyar** [comment]
I don't think that's necessary as there are no IP implications, it would only clutter the readme with unnecessary details

File: README.md [hunk 1]
```diff
@@ -1,211 +1,20 @@
...
-* See demos here (under heavy development): https://github.com/ros-controls/ros2_control_demos
 
-`ros2_control` is a proof of concept on how new features within ROS 2 can be elaborated and used in the context of robot control.
-We were keeping the vocabulary close the ROS 1 implemenation, however this is with no notion a migration, but rather a fresh new-write.
-We hope that this can be a starting point for migrating ros_control to ROS 2 at some point.
-Reasons for re-writing this repo from scratch rather than porting existing code to ROS 2 syntax is to leverage the new ROS 2 concepts in full.
-The work done in this repo (together with [`ros2_controllers`](https://github.com/ros-controls/ros2_controllers)) was presented at [ROSCon 2017](https://vimeo.com/236182180) and is currently not under active development.
+## About
+This package is a part of the ros2_control framework. 
+For further description check the [documentation](https://github.com/ros-controls/control.ros.org).
```
Comment (v-lopez) [inline README.md:14]: Shouldn't it point to https://ros-controls.github.io/control.ros.org/ and later to https://control.ros.org/?

If someones is reading the readme, probably they just want the doc, not the source files for the doc.

Comment (bmagyar) [reply README.md:14]:
```suggestion
For further description check the [documentation](https://ros-controls.github.io/control.ros.org/).
```

File: README.md [hunk 2]
```diff
@@ -1,211 +1,20 @@
...
-* See demos here (under heavy development): https://github.com/ros-controls/ros2_control_demos
 
-`ros2_control` is a proof of concept on how new features within ROS 2 can be elaborated and used in the context of robot control.
-We were keeping the vocabulary close the ROS 1 implemenation, however this is with no notion a migration, but rather a fresh new-write.
-We hope that this can be a starting point for migrating ros_control to ROS 2 at some point.
-Reasons for re-writing this repo from scratch rather than porting existing code to ROS 2 syntax is to leverage the new ROS 2 concepts in full.
-The work done in this repo (together with [`ros2_controllers`](https://github.com/ros-controls/ros2_controllers)) was presented at [ROSCon 2017](https://vimeo.com/236182180) and is currently not under active development.
+## About
+This package is a part of the ros2_control framework. 
+For further description check the [documentation](https://ros-controls.github.io/control.ros.org/).
```
Comment (bmagyar) [inline README.md:14]:
```suggestion
For more, please check the [documentation](https://ros-controls.github.io/control.ros.org/).
```

**Review by bmagyar — APPROVED** [review]

**Pull request merged**